src/common/Database/MySQLConnection.cpp

```diff
@@ -107,36 +112,59 @@ bool MySQLConnection::Open()
     }
     #endif
 
-    m_Mysql = mysql_real_connect(mysqlInit, m_connectionInfo.host.c_str(), m_connectionInfo.user.c_str(),
-        m_connectionInfo.password.c_str(), m_connectionInfo.database.c_str(), port, unix_socket, 0);
-
-    if (m_Mysql)
-    {
-        if (!m_reconnecting)
+    // Possible improvement for future: make ATTEMPTS and SECONDS configurable values
+    uint const ATTEMPTS = 180;
+    uint const SECONDS = 10;
+
+    uint count = 0;
+    do {
+        m_Mysql = mysql_real_connect(
```

Comment:
why you split it line by line in string more easy to read i think

Reply:
too-long lines are not good, better to split for readability :)

Reply:
ok no problem just ask